hashtagmap-kakao-scheduler/src/main/java/com/songpapeople/hashtagmap/scheduler/domain/CronPeriod.java

```diff
@@ -10,15 +12,24 @@
     private Trigger trigger;
 
     public CronPeriod(String expression) {
+        validateCronTrigger(expression);
         this.trigger = new CronTrigger(expression, TIME_SEOUL);
```

Comment:
아니면 이부분 전체를 try catch로 잡아두어도 괜찮을것 같아요
```suggestion
        try {
            this.trigger = new CronTrigger(expression, TIME_SEOUL);
        } catch (IllegalArgumentException e) {
            throw new KakaoSchedulerException(KakaoSchedulerExceptionStatus.INVALID_PERIOD_EXPRESSION);
        }
```